repos.d/rpm/amazonlinux.yaml

```diff
@@ -0,0 +1,33 @@
+- name: amzn2
+  type: repository
+  desc: Amazon Linux 2
+  family: fedora
```

Comment:
I'd like a comment on that: is it known to follow fedora package naming style (e.g. not centos/mandriva/mageia/suse/...)? Maybe it's known to be based on specific distro?

Reply:
it is closest to centos though some packages are derived from fedora.  Will switch to centos.